src/renderer/content.ts

```diff
@@ -1,4 +1,112 @@
-import { EditorId } from '../interfaces';
+import { appState } from './state';
+import { USER_DATA_PATH } from './constants';
+import {
+  EditorId,
+  EditorValues,
+  RunnableVersion,
+  VersionSource,
+} from '../interfaces';
+import { fancyImport } from '../utils/import';
+import { readFiddle } from '../utils/read-fiddle';
+
+import * as fetchType from 'node-fetch';
+import * as fsType from 'fs-extra';
+import * as path from 'path';
+import * as semver from 'semver';
+import decompress from 'decompress';
+
+// parent directory of all the downloaded template fiddles
+const TEMPLATES_DIR = path.join(USER_DATA_PATH, 'Templates');
+
+// location of the fallback template fiddle used iff downloading failed
+const STATIC_TEMPLATE_DIR = path.resolve(
+  __dirname,
+  '../../static/electron-quick-start',
+);
+
+/**
+ * Ensure we have a fiddle for the specified Electron branch.
+ * If we don't have it already, download it from electron-quick-start.
+ *
+ * @param {string} branch Electron branchname, e.g. `12-x-y` or `master`
+ * @returns {Promise<string>} Path to the folder where the fiddle is kept
+ */
+async function prepareTemplate(branch: string): Promise<string> {
+  let folder = path.join(TEMPLATES_DIR, `electron-quick-start-${branch}`);
+
+  try {
+    // if we don't have it, download it
+    const fs = await fancyImport<typeof fsType>('fs-extra');
+    if (!fs.existsSync(folder)) {
+      console.log(`Content: Downloading template for ${branch}`);
+      const url = `https://github.com/electron/electron-quick-start/archive/${branch}.zip`;
+      const fetch = (await fancyImport<any>('node-fetch')).default;
```

Comment:
Great question! I did it to be able to use a node Buffer, but since you asked, I checked and now see I can get an ArrayBuffer from the native fetch and then use `Buffer.from()` to convert it to a node Buffer.

node-fetch removed in 402921d8630bcb184c5a0e4a02eaba4eebffc5b5